components/form/UploadImageInput.tsx

```diff
@@ -114,19 +117,17 @@ const UploadImageInput: React.FC<UploadImageProps> = ({
 
   return (
     <div className='my-3 flex flex-col items-center gap-4'>
-      <label htmlFor='image'>Upload an image:</label>
+      <label htmlFor='image' className='flex items-center gap-1'>
+        Upload an image
+        {isRequired && <span className=''>*</span>}
+      </label>
       <input
         className='pl-14'
         type='file'
         name='image'
         onChange={(e) => imageFileUpload(e)}
-        required={isRequired}
```

Comment:
Tested it out and I can now actually upload an item without an image. The label will show under images if some other field is not filled in but if I fill in the rest of the information I can submit without an image

Reply:
This has been resolved in the last commit and now all input fields in the form use the same "required" hook for validation and custom error message.